collectors/builtin/services_startup.py

```diff
@@ -60,19 +68,25 @@ def __call__(self):
     def process(self, line):
         for service in self.services:
             service = service.strip()
-	    if service in line:
+            if service in line:
+                if self.initialize:
+                    self.services_before.append(service)
```

Comment:
These two lines for the initial case are wrong. We are not able to detect a service which stops before the collector starts. For example, if there are A and B in config, after collector starts, A is detected to be running but B is not, then services_before = [A] = services_after. Thus, you are not able to tell if B is stopped.